### DISCUSSION TITLE
After the update, the port cannot be activated

### DISCUSSION THREAD

**Discussion by exan0313** [thijsvanloef/palworld-server-docker#574] [Q&A] [answered]
I'm asking for help with a problem I don't understand.
After updating the version of the container in the last few days, I restarted the docker and found that my service was up, but the port was not enabled!
<img width="715" alt="2024-07-03 10 48 32" src="https://github.com/thijsvanloef/palworld-server-docker/assets/43337047/e9393efb-1870-4c63-87e0-450bd4df1329">

I also confirmed the settings in docker-compose.yml.
<img width="821" alt="截圖 2024-07-03 晚上10 51 24" src="https://github.com/thijsvanloef/palworld-server-docker/assets/43337047/94f48c9e-d6e7-4c43-abdb-d89e3e62b563">

I can't figure out why docker has enabled port 8211 and 27015, but the status is not LISTEN.
I would like to ask you how to solve this problem.

**Comment by exan0313** [comment]
Additional information: I have also checked the port status in .env and it's working fine.
<img width="340" alt="2024-07-03 10 56 52" src="https://github.com/thijsvanloef/palworld-server-docker/assets/43337047/ded9b5bc-e936-4aea-80d4-cfc1b6db12f5">

**Comment by MusclePr** [comment, marked as answer]
UDP is a stateless protocol.
